packages/components/src/Field-Next/Label.tsx

```diff
@@ -36,15 +36,15 @@ export const Label: React.FC<LabelProps> = ({
     <Box
       {...props}
       as={as}
-      variant="label"
+      variant="label.above"
```

Comment:
Yes, because it's not "above" but a label (unlike what we use in checkbox. We use the element there but only because we need the functionality) and as sad in the PR description: it's temporary so that they new one looks like the old one